spec/classes/init_spec.rb

```diff
@@ -473,6 +473,8 @@
     it { should contain_file('sshd_config').with_content(/^HostKey \/etc\/ssh\/ssh_host_rsa_key/) }
     it { should contain_file('sshd_config').with_content(/^HostKey \/etc\/ssh\/ssh_host_dsa_key/) }
     it { should contain_file('sshd_config').with_content(/^StrictModes yes$/) }
+    it { should_not contain_file('sshd_config').with_content(/^PermitUserEnvironment/) }
```

Comment:
These are in the section 'with params used in sshd_config set on valid osfamily' so that means they should be set to a valid, non-default value.